src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -473,10 +473,11 @@ public V getRaw(ResourceLocation key)
         return ret;
     }
 
-    void addAlias(ResourceLocation from, ResourceLocation to)
+    // TODO: Add as public api in IForgeRegistry - currently visible for remapping reach/range attributes.
```

Comment:
Toss a @ApiStatus.Internal onto the entire ForgeRegistry class. People should be accessing it through our public API.